tensorflow/tensorflow_plugin.cc

```diff
@@ -974,7 +985,11 @@ struct TensorflowGraphBase: public Function {
             }
         }
         else if (val.isEmbedding()) {
-            // TODO: infer consistent type for tensor, and convert it
+            auto storageType = val.getEmbeddingType();
+            if (storageType == ST_ATOM) {
+                storageType = ST_FLOAT64; //TODO not sure its possible to get consistent type then
```

Comment:
I see it as a big improvement over the previous TODO! I'll add something to tensorflow.md, not quite sure how to word it.